src/Main.java

```diff
@@ -0,0 +1,33 @@
+import java.util.Scanner;
+/*
+* https://stackoverflow.com/questions/2066307/how-do-you-input-commandline-argument-in-intellij-idea
+* */
+public class Main{
+
+    public static void main(String[] args){
+
+        // To keep the user input (insert o query statements)
+        //String userInput;
+        // We instanciate a new Scanner
+        //Scanner scanner = new Scanner(System.in);
+        /*System.out.println("Welcome to databasic\n====================");
+        System.out.println("Type a command:");
+        userInput = scanner.nextLine();*/
+
+        // Check if input isn't empty
+        InputProcessor inputProcessor = new InputProcessor();
```

Comment:
I really like that you delegated to another class here!
For reasons that I'll get into later, it's a good idea to create a new object in `main()` and have your application somewhere other than `main`.

As a rule of thumb, I try and always make my `main` method be 1 line long. You can do this too, if you do: 

```java
new InputProcessor().validateInput(args)
```

Obviously, this won't be enough once we start writing more and more of the application. So in the future we would probably do something like:

```java
new Application().start(args)
```

Where `Application` is a class that handles validating input, delegating to insert and query methods, etc.

Reply:
Done, thanks!